src/test/java/ai/rev/speechtotext/integration/EnvHelper.java

```diff
@@ -3,6 +3,6 @@
 public class EnvHelper {
 
     public static String getToken() {
-        return System.getenv("TOKEN");
+        return System.getenv("REVAI_ACCESS_TOKEN");
```

Comment:
the less-generic name makes it more convenient to test locally without possible name clashes with other programs for the variable